cmake/FindOpus.cmake

```diff
@@ -15,6 +15,27 @@ find_library(Silk_LIBRARY NAMES silk_common
 
 find_package_handle_standard_args(Opus REQUIRED_VARS Opus_INCLUDE_DIR Opus_LIBRARY)
 
+if(NOT TARGET Opus::opus)
+    add_library(Opus::opus UNKNOWN IMPORTED)
+    set_target_properties(
+        Opus::opus PROPERTIES
+        INTERFACE_INCLUDE_DIRECTORIES ${Opus_INCLUDE_DIR}
+        IMPORTED_LOCATION ${Opus_LIBRARY}
+    )
+    if(EXISTS ${Celt_LIBRARY})
+        set_property(
+            TARGET Opus::opus APPEND PROPERTY
+            INTERFACE_LINK_LIBRARIES ${Celt_LIBRARY}
+        )
+    endif()
+    if(EXISTS ${Silk_LIBRARY})
+        set_property(
+            TARGET Opus::opus APPEND PROPERTY
+            INTERFACE_LINK_LIBRARIES ${Cilk_LIBRARY}
```

Comment:
Cilk, while pronounced the same, is not a good substitute for Silk.
:trollface:

Reply:
Good thing all that cilk mess is very, very old opus :tongue: